src/pygambit/behav.pxi

```diff
@@ -324,7 +324,36 @@ class MixedBehaviorProfile:
             If `action` is a string and no action in the game has that label.
         """
         return self._action_value(self.game._resolve_action(action, 'action_value'))
-
+	
+    def realiz_prob(self, player: typing.Union[Player, str], action: Action, node: Node) -> float:
```

Comment:
A realisation probability is a property of a node only.  The function we're calling in C++ is src/games/behav.h:174:

```
  const T &GetRealizProb(const GameNode &node) const;
```

So the argument should just be a `node: typing.Union[Node, str]` (because we want the usual `_resolve_node` mechanism to work for this)